include/common_structs.h

```diff
@@ -104,7 +104,7 @@ typedef struct Npc {
     /* 0x08A */ s16 yawCamOffset;
     /* 0x08C */ char unk_8C[2];
     /* 0x08E */ s16 duration; /* formerly interp_counter */
-    /* 0x090 */ s16 homePos[3];
+    /* 0x090 */ Vec3s homePos;
```

Comment:
Maybe drop "pos"? Position is kinda implied by the `Vec` type and hungarian notation bad. Same with other fields but that's out-of-scope for this PR.

Reply:
I feel like simply "home" is a poor name for what is presumably some sort of default position.